src/GraphQL/Internal/Name.hs

```diff
@@ -17,13 +18,67 @@ module GraphQL.Internal.Name
 
 import Protolude
 
+import qualified Data.Aeson as Aeson
 import GHC.TypeLits (Symbol, KnownSymbol, symbolVal)
-import GraphQL.Internal.Syntax.AST
-  ( Name(..)
-  , NameError(..)
-  , unsafeMakeName
-  , makeName
-  )
+import Data.Char (isDigit)
+import Data.Text as T (Text)
+import qualified Data.Attoparsec.Text as A
+import Test.QuickCheck (Arbitrary(..), elements, listOf)
+import Data.String (IsString(..))
+import Data.Text as T (Text, append, empty)
```

Comment:
Don't need to import `Text` here as it's already imported four lines above.

Reply:
Fixed. Waiting on `Name ""` vs `Nothing` to push.